src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -229,6 +229,15 @@ class DefaultStorageKeyAllocator
     static StorageKeyName TSDefaultNTP() { return StorageKeyName::FromConst("g/ts/dntp"); }
     static StorageKeyName TSTimeZone() { return StorageKeyName::FromConst("g/ts/tz"); }
     static StorageKeyName TSDSTOffset() { return StorageKeyName::FromConst("g/ts/dsto"); }
+
+    static StorageKeyName FabricICDClientInfoCounter(FabricIndex fabric) { return StorageKeyName::Formatted("f/%x/icdc", fabric); }
+
+    static StorageKeyName ICDClientInfoKey(FabricIndex fabric) { return StorageKeyName::Formatted("f/%x/icdk", fabric); }
+
+    // ICDFabricList is only used by DefaultICDClientStorage
+    // when new fabric is created, this list needs to be updated,
+    // when client init DefaultICDClientStorage, this table needs to be loaded.
```

Comment:
_All three_ keys are only used in DefaultICDClientStorage, no?

Reply:
Yes, would update them in follow-up PR, thanks